drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c

```diff
@@ -1949,6 +1963,8 @@ int vchiq_mmal_init(struct vchiq_mmal_instance **out_instance)
 
 err_close_services:
 
+	if (instance->bulk_wq)
```

Comment:
Depending on the answer to an earlier question you may want to remove this condition.